server/db/models/user.js

```diff
@@ -63,6 +106,11 @@ const setSaltAndPassword = user => {
   }
 }
 
+const validatePhone = user => {
+  user.phone = +user.phone.replace(/[-]/g, '')
```

Comment:
Just add replacements for characters like parentheses, dots, and spaces. 
Or just make the whole regex swap out any non-numeric?